library/pom.xml

```diff
@@ -378,6 +378,12 @@
         </exclusion>
       </exclusions> 
     </dependency>
+    <dependency>
```

Comment:
Shouldn't there be also dependencies removed as they are replaced by the mock server?